docresources/VersionBot/versionbot.md

```diff
@@ -115,7 +113,94 @@ procbot:
+
+````
+minimum_approvals: <number>
+reviewers:
+    - <list of Github user names>
+maintainers:
+    - <list of Github user names>
+````
+
+* `minimum_approvals`[optional] - Denotes the minimum number of approvals required for a PR
+* `reviewers`[optional] - A list of Github user names whose reviews count towards an approval total
+* `maintainers`[optional] - A list of Github user names whose reviews count towards an approval total, of which at *least* one must be a maintainer in this list
+
+The functionality is as follows:
+* If a `reviewers` list exists, then reviews counting towards `minimum_approvals`
+    can only be met by reviewers in that list (and `maintainers`)
+* If a `maintainers` list exists, then at least one review must be from a maintainer:
+    * If a maintainer created the PR *and* there are no other maintainers, then reviews
+    from `reviewers` (if it exists) are required to make up the `minimum_approvals`
+    * If a maintainer created the PR and there *are* other maintainers, then one of the
+    remaining `maintainers` must approve the PR
+* If no `reviewers` list exists, anyone can approve; if a `maintainers` list exists
+    then at least one `maintainer` must approve
+* For any named reviewer/maintainers not added when a PR is opened, VersionBot will
+    request that they add themselves on the PR as reviewers
+
+Example:
+
+```
+minimum_approvals: 1
+reviewers:
+    - githubuser
+maintainers
+    - githubmaintainer
+```
+
+#### Required Tags in PR Commits
+
+This functionality allows repository maintainers to determine which footer tags in
+a PR are mandatory or optional.
+Tags keys are considered case insensitive, and optionally may define how many
+occurences should be present in all of the commits for a PR, as well as defining
+valid values.
+
+The `change-type` tag is *always* assumed to be present, regardless of other
+tag definitions. Should the configuration file redefine the `change-type` tag,
+then that definition will override the inbuilt one.
+
+The following shows how to format required tag definitions in the `repository.yml`
+configuration:
+
+```
+required_tags:
+    - name: <tagname>
+        occurrence: <number> | 'all'
+        values: <regexp>
+```
+
+Both the `occurrence` and `values` properties are optional, if present:
+* `occurence`[optiona] - Specifies the minimum number of commits in the PR that must contain
+                the tag. 'all' means every commit in the PR. Should the number of
+                commits in a PR be less than this number, it is assumed that they
+                are valid (as long as those commits contain the tag).
+                Defaults to 1 should it not be defined
+* `values`[optional] - A regular expression defining a tags valid values. This regexp should
+            be defined as if the entire line containing the tag were present, but
+            should only look for valid values.
+            Defaults to /.*$/ should it not be defined.
```

Comment:
Technically, it defaults to `/.*/` (no `$`). Just being picky though, it doesn't make any difference.